crates/astria-bridge-withdrawer/src/withdrawer/mod.rs

```diff
@@ -97,28 +123,52 @@ impl BridgeService {
         });
         info!("spawned API server");
 
+        let mut submitter_task = tokio::spawn(submitter.run());
+        info!("spawned submitter task");
         let mut ethereum_watcher_task = tokio::spawn(ethereum_watcher.run());
         info!("spawned ethereum watcher task");
 
         let shutdown = select!(
             o = &mut api_task => {
                 report_exit("api server", o);
-                Shutdown { api_task: None, ethereum_watcher_task: Some(ethereum_watcher_task), api_shutdown_signal, token: shutdown_token }
+                Shutdown {
+                    api_task: None,
+                    submitter_task: Some(submitter_task),
+                    ethereum_watcher_task: Some(ethereum_watcher_task),
+                    api_shutdown_signal,
+                    shutdown_token
+                }
+            }
+            o = &mut submitter_task => {
+                report_exit("submitter", o);
+                Shutdown {
+                    api_task: Some(api_task),
+                    submitter_task: None,
+                    ethereum_watcher_task:Some(ethereum_watcher_task),
+                    api_shutdown_signal,
+                    shutdown_token
+                }
             }
             o = &mut ethereum_watcher_task => {
                 report_exit("ethereum watcher", o);
-                Shutdown { api_task: Some(api_task), ethereum_watcher_task: Some(ethereum_watcher_task), api_shutdown_signal, token: shutdown_token }
+                Shutdown {
+                    api_task: Some(api_task),
+                    submitter_task: Some(submitter_task),
+                    ethereum_watcher_task: None,
+                    api_shutdown_signal,
+                    shutdown_token
+                }
             }
 
         );
         shutdown.run().await;
     }
 }
 
-/// A handle for instructing the [`Bridge`] to shut down.
+/// A handle for instructing the [`BridgeService`] to shut down.
```

Comment:
```suggestion
/// A handle for instructing the [`WithdrawerService`] to shut down.
```

same in rest of comment